### PR TITLE
Bumping go version to v1.19.6 in v0.44.x line

### PR DESCRIPTION
**Pull request by sethiyash**
- Bumping go version to v1.19.6 in v0.44.x line

### CODE REVIEW

**Comment by vmunishwar** [comment]
LGTM

File: .github/workflows/golangci-lint.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
       - name: Install Go 1.19.5
```
Comment (vmunishwar) [inline .github/workflows/golangci-lint.yml]: This can be either Install Go or Install Go 1.19.6

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Install Go 1.19.5
```
Comment (vmunishwar) [inline .github/workflows/release.yml]: This can be either Install Go or Install Go 1.19.6

File: .github/workflows/test-all.yml [hunk 3]
```diff
@@ -13,7 +13,7 @@ jobs:
     - name: Install Go 1.19.5
```
Comment (vmunishwar) [inline .github/workflows/test-all.yml]: This can be either Install Go or Install Go 1.19.6

File: .github/workflows/test-windows.yml [hunk 4]
```diff
@@ -13,7 +13,7 @@ jobs:
     - name: Install Go 1.19.5
```
Comment (vmunishwar) [inline .github/workflows/test-windows.yml]: This can be either Install Go or Install Go 1.19.6

**Review by vmunishwar — CHANGES_REQUESTED** [review]
Please update the name to reflect correct version -  Install Go 1.19.6

File: .github/workflows/test-windows.yml (hunk 4 above)
Comment (sethiyash) [reply .github/workflows/test-windows.yml]: Adopted.

File: .github/workflows/test-all.yml (hunk 3 above)
Comment (sethiyash) [reply .github/workflows/test-all.yml]: Adopted.

File: .github/workflows/release.yml (hunk 2 above)
Comment (sethiyash) [reply .github/workflows/release.yml]: Adopted.

File: .github/workflows/golangci-lint.yml (hunk 1 above)
Comment (sethiyash) [reply .github/workflows/golangci-lint.yml]: Adopted.

**Review by vmunishwar — APPROVED** [review]
LGTM

**Pull request merged**